_posts/2021-07-01-wildfly-preview-bootable-jar.adoc

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
+title: Run WildFly Preview of Jakarta EE 9.1 with Bootable Jar
+date: 2021-07-01
+tags: wildfly jakarta-ee9 bootable-jar cloud
+synopsis: This article demonstrates how to run a Jakarta EE 8 application on WildFly Preview for Jakarta EE 9.1 using the Bootable Jar.
+author: jmesnil
+---
+
+https://jakarta.ee/[Jakarta EE 9.1] has been recently released and WildFly 24.0.0.Final provides a preview to run existing Jakarta EE 8 applications on it.
```

Comment:
@jmesnil https://jakarta.ee/specifications/platform/9.1 might be a better link for `Jakarta EE 9.1`,  as `jakarta.ee` contains much more than just EE 9.1.

Reply:
fixed, thanks